docs/troubleshooting.md

```diff
@@ -44,3 +44,4 @@ Your system has cmdtest installed, which provides a different program as yarn. U
 - **To check if redis is running as a daemon in Linux** `ps aux | grep redis-server`
 
 - Use node v10 or lower to avoid any errors.
+- **For WSL users , if rspec tests are taking too long to run** make sure to fork the repo in the linux file system and not in the windows partition. Use command `pwd` to know the exact location of your repo.
```

Comment:
This could be more specific in terms of how to move the files or re-download them in the linux FS. A link to documentation about this general issue might also be appropriate.